R/contingencytables.R

```diff
@@ -794,11 +810,38 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
         proportions.matrix[,] <- ""
       }
 
+      residuals.matrix <- try({
+        counts.matrix - stats::chisq.test(counts.matrix, correct = FALSE)$expected
+      })
```

Comment:
It would perhaps make sense to run

```r
chisqResults <- try(stats::chisq.test(counts.matrix, correct = FALSE))
```

earlier and then just extract the `$expected`, `$residuals`, `$stdres` from the `chisqResults` object without rerunning the test every time.